component-runtime-manager/src/test/java/org/talend/sdk/component/runtime/manager/ParameterModelServiceTest.java

```diff
@@ -111,6 +112,54 @@ void charAsString() throws NoSuchMethodException {
         }
     }
 
+    @Test
+    void intAsLongByDefault() throws NoSuchMethodException {
```

Comment:
Why the name of the method is '..AsLong', seems like you are testing int vs Integer constraints here

Reply:
yes, you are right. Originally it was another idea for this test. I'll rename it. Thank you.